pyinjective/denoms_mainnet.ini

```diff
@@ -430,6 +430,15 @@ min_display_price_tick_size = 9.999999999999999e-05
 min_quantity_tick_size = 100000
 min_display_quantity_tick_size = 0.1
 
+[0x35fd4fa9291ea68ce5eef6e0ea8567c7744c1891c2059ef08580ba2e7a31f101]
+description = 'Mainnet Spot TIA/USDT'
+base = 6
+quote = 6
+min_price_tick_size = 0.001
+min_display_price_tick_size = 9.999999999999999e-05
```

Comment:
could it be 0.001? (although it won't effect API result)

Reply:
Sorry @vinhphuctadang, I had to release a new version of the SDK including the market in a rush, and I did not get to see your comment on time.
I will apply the change in a separate PR tomorrow my time

Reply:
no worries, I checked it won't effect trading API so please help when you're up, thank you